torch/csrc/api/include/torch/nn/module.h

```diff
@@ -108,31 +215,106 @@ class Module {
   void load(Archive& ar);
 
   /// Attempts to cast this `Module` to the given `ModuleType`.
+  ///
+  /// This method is useful when calling `apply()` on a `ModuleCursor`.
+  /// \rst
```

Comment:
Other idle wondering: should we just turn on ReST by default?

Reply:
ReSt is so cumbersome to read and write compared to Markdown IMO, just like how code has to be double backticked for example